### DISCUSSION TITLE
CDS Labels for multi-contig GenBank file?

### DISCUSSION THREAD

**Discussion by acarafat** [moshi4/pyCirclize#38] [General]
Hello,

First thank you for working on this project to make circos plot super easy. 

I'm working on a multi-contig GBK file right now.

Is there an easy way to plot the CDS labels for each sectors representing separate contigs? 

Labels from `gbk.extract_features` gives labels from all contigs. Having something that is specific to the contigs/sectors would be useful. 

Let me know what you think.

**Comment by moshi4** [comment]
You can use `gbk.get_seqid2features()` instead of `gbk.extract_features()`.

Code examples can be found in the following document.
https://moshi4.github.io/pyCirclize/circos_plot/#3-mycoplasma-alvi

**Reply by acarafat** [reply]
Thanks. I have used the suggested `gbk.get_seqid2features()` method: looping though the sectors individually, and pulling the seqFeature labels & position, finally plotting them.
